### PR TITLE
Add help (#113)

### PR DESCRIPTION
**Pull request by skuzzymiglet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ In most cases this command must be run by root user or through `sudo` command.
 
 #### Building for Linux i386
 
-Along with `GOARCH`, set `CGO_ENABLED=1`. If you're on amd64, make sure to install 32-bit libasound. On Arch, enable multilib by uncommenting the [multilib] section in `/etc/pacman.conf`
+Along with `GOARCH`, set `CGO_ENABLED=1` (see [here](https://golang.org/cmd/cgo/#hdr-Using_cgo_with_the_go_command)). Make sure to install the shared libraries for the target platform.
```
Comment (hajimehoshi) [inline README.md:45]: * Along with `GOOS` or `GOARCH`
* s/here/the official document about Cgo/

**Comment by skuzzymiglet** [comment]
OK I came back to this PR. It's now "cross-compiling" not "i386"

File: README.md [hunk 2]
```diff
@@ -40,6 +40,10 @@ apt install libasound2-dev
 
 In most cases this command must be run by root user or through `sudo` command.
 
+#### Crosscompiling
+
+To crosscompile, make sure the libraries for the target architecture are installed, and set `CGO_ENABLED=1` as Go disables CGO on crosscompiles by default
```
Comment (hajimehoshi) [inline README.md:45]: s/CGO/[Cgo](https://golang.org/cmd/cgo/#hdr-Using_cgo_with_the_go_command)/

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**